### PR TITLE
Ignore latency metrics for events

### PR DESCRIPTION
**Pull request by fgrzadkowski**
This should fix some of the jenkins failure.

@wojtek-t @bprashanth @quinton-hoole

### CODE REVIEW

**Comment by fgrzadkowski** [comment]
See #7561

File: test/e2e/util.go [hunk 1]
```diff
@@ -761,14 +761,20 @@ func ReadLatencyMetrics(c *client.Client) ([]LatencyMetric, error) {
...
 // Prints summary metrics for request types with latency above threshold
 // and returns number of such request types.
-func HighLatencyRequests(c *client.Client, threshold time.Duration) (int, error) {
+func HighLatencyRequests(c *client.Client, threshold time.Duration, ignoredResources []string) (int, error) {
 	metrics, err := ReadLatencyMetrics(c)
 	if err != nil {
 		return 0, err
 	}
+	ignored := make(map[string]bool)
+	for _, resource := range ignoredResources {
```
Comment (bprashanth) [inline test/e2e/util.go]: suggest declaring this as a util.StringSet and checking the set with Has, instead of a list that becomes a dict

**Comment by bprashanth** [comment]
LGTM with one nit

File: test/e2e/util.go (hunk 1 above)
Comment (wojtek-t) [inline test/e2e/util.go]: +1

Comment (fgrzadkowski) [inline test/e2e/util.go]: Done.

**Comment by wojtek-t** [comment]
LGTM - will merge on green.

**Pull request merged**